step1__select_samples.py

```diff
@@ -83,18 +83,28 @@ def main(args):
     logger.info(
         f"Taking up to {args.num_samples} samples per site where samples are het, hom_var, or hemi"
     )
+
+    def sample_ordering_expr(mt):
+        # It can be problematic for downstream steps when several samples have many times more variants selected
```

Comment:
just out of curiosity, why did you choose to use comments here rather than a docstring?

Reply:
not sure